docs/source/etl/variant-data.rst

```diff
@@ -182,6 +182,11 @@ The schema of the resulting DataFrame matches that of the VCF reader.
 | ``hardCallThreshold`` | double  | 0.9          | Sets the threshold for hard calls.                                                                         |
 +-----------------------+---------+--------------+------------------------------------------------------------------------------------------------------------+
 
+.. important::
+
+    The BGEN reader and writer assume that the first allele is the reference allele, and that all following alleles are
```

Comment:
Just for more clarity I would write "... assume that in the .bgen file the first allele is..." instead of "assume that the fist allele is..."